packages/amplication-server/src/core/git/dto/inputs/CreateGitRepositoryInput.ts

```diff
@@ -1,12 +1,44 @@
 import { Field, InputType } from "@nestjs/graphql";
-import { CreateGitRepositoryBaseInput } from "./CreateGitRepositoryBaseInput";
+import { EnumGitOrganizationType } from "../enums/EnumGitOrganizationType";
+import { EnumGitProvider } from "../enums/EnumGitProvider";
 
 @InputType({
   isAbstract: true,
 })
-export class CreateGitRepositoryInput extends CreateGitRepositoryBaseInput {
+export class CreateGitRepositoryInput {
```

Comment:
I see in other files that we are still using `CreateGitRepositoryBaseInput` so I just want to make sure you didn't miss anything